src/main/java/jalau/at18/kingoftokyo/RandomDice.java

```diff
@@ -0,0 +1,28 @@
+package jalau.at18.kingoftokyo;
+
+import java.util.Random;
+
+public class RandomDice {
+    private int randomDice;
+
+    public Dice generatedRandom() {
+        randomDice = new Random().nextInt(DiceFace.values().length) + 1;
+        for (DiceFace element : DiceFace.values()) {
```

Comment:
I think you don't need a `for` block for this, you could simple get the `DiceFace` element from values in the random position, then created the Dice with that value
```
DiceFace face = DiceFace.values()[randomDice];
Dice dice = new Dice(face);
```

Reply:
I will update the code